app/views/home/_create_activity.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with(url: activity_index_path, method: :post) do |f| %>
```

Comment:
why don't we use a model for the activity here?

Reply:
this would look something like `<%= form_with(model: Activity.new) do |f| %>`

Reply:
Doesn't seem to be that simple